fuse_constraints/include/fuse_constraints/util.h

```diff
@@ -59,6 +59,14 @@ void wrapAngle2D(T& angle)
   angle -= TAU * ceres::floor((angle + PI) / TAU);
 }
 
+template<typename T>
+T wrapAngle2D(const T& angle)
+{
+  T wrapped = angle;
```

Comment:
`T wrapped(angle);`
`T wrapped = T(angle);`
???
Trying to decide how our policy of explicit casting should be implemented here. Or even if any of these options are actually different.

Reply:
`T wrapped { angle }` may be better, since it doesn't permit narrowing.